src/zodbbrowser/browser.py

```diff
@@ -301,10 +389,11 @@ def getUrl(self, oid=None, tid=None):
         if oid is None:
             oid = self.getObjectId()
         url = "@@zodbbrowser?oid=0x%x" % oid
-        if tid is None and 'tid' in self.request:
-            url += "&tid=" + self.request['tid']
-        elif tid is not None:
-            url += "&tid=0x%x" % tid
+        if self.supportsVersions:
```

Comment:
This "supportsVersions" business seems mistaken.

AFAIU you're trying to distinguish storages that support history from storages that don't.  "supportsVersions" is not related to history, so this kind of checking incorrectly turns off history browsing for FileStorage.